.gitignore

```diff
@@ -22,6 +22,14 @@
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
 
+<<<<<<< HEAD
```

Comment:
Whenever you see lines with `<<<<<<<` it's because there has been a merge conflict in the file. This kind of lines should be deleted 😋 
When you have merge conflicts git automatically adds something like this to the conflicted files:
`<<<<<<< HEAD`
`   the code you have on your branch (the one you've been working on)`
`=======`
`   the code that's on the branch you are trying to merge with (in this case gf-branch-3)`
`>>>>>>> gf-branch-3`
You have to choose which code is the one that has to stay and erase the rest.

```suggestion
```